ArmPlatformPkg/Include/Library/ArmPlatformLib.h

```diff
@@ -136,4 +136,24 @@ ArmPlatformGetPlatformPpiList (
   OUT EFI_PEI_PPI_DESCRIPTOR  **PpiList
   );
 
+// MU_CHANGE START: Allow platform to customize initial memory region.
+
+/**
+  Checks if the platform requires a special initial EFI memory region.
+
+  @param[out]  EfiMemoryBase  The custom memory base, should be unchanged if FALSE is returned.
+  @param[out]  EfiMemorySize  The custom memory size, should be unchanged if FALSE is returned.

```

Comment:
nit: should be -> will be (I like strong contracts :) if that isn't true, then I would call that out specifically.

Reply:
Same below.